docs/source/contributing/community.rst

```diff
@@ -4,27 +4,19 @@
 Community communication channels
 ================================
 
-We use `Discourse <https://discourse.jupyter.org>` for online discussion.
-Everyone in the Jupyter community is welcome to bring ideas and questions there.
-In addition, we use `Gitter <https://gitter.im>`_ for online, real-time text chat,
-a place for more ephemeral discussions.
-The primary Gitter channel for JupyterHub is `jupyterhub/jupyterhub <https://gitter.im/jupyterhub/jupyterhub>`_.
-Gitter isn't archived or searchable, so we recommend going to discourse first
-to make sure that discussions are most useful and accessible to the community.
-Remember that our community is distributed across the world in various
-timezones, so be patient if you do not get an answer immediately!
+There is a place for everyone in the Jupyter community to share thoughts and ask questions. We use `Discourse <https://discourse.jupyter.org>`_ , `Gitter <https://gitter.im>`_ , and GitHub issues for online discussion.
 
-GitHub issues are used for most long-form project discussions, bug reports
-and feature requests. Issues related to a specific authenticator or
-spawner should be directed to the appropriate repository for the
-authenticator or spawner. If you are using a specific JupyterHub
+JupyterHub's main Gitter channel is `jupyterhub/jupyterhub <https://gitter.im/jupyterhub/jupyterhub>`_.
+However, to ensure that discussions are the most useful and accessible to the community, we advise using Discourse first as Gitter doesn't have an archive or search function
+Please be patient if you do not recieve a response right away; keep in mind that our comunity is spread out over the globe in different timezones.
+
+Most lengthy project conversations, bug reports, and feature requests take place via GitHub issues. If you have an issue with a certain authenticator or spawner, you should report them to the relevant repository. If you are using a specific JupyterHub
 distribution (such as `Zero to JupyterHub on Kubernetes <http://github.com/jupyterhub/zero-to-jupyterhub-k8s>`_
 or `The Littlest JupyterHub <http://github.com/jupyterhub/the-littlest-jupyterhub/>`_),
-you should open issues directly in their repository. If you can not
+you should open issues directly in the respective repository. If you can not
 find a repository to open your issue in, do not worry! Create it in the `main
 JupyterHub repository <https://github.com/jupyterhub/jupyterhub/>`_ and our
 community will help you figure it out.
 
-A `mailing list <https://groups.google.com/forum/#!forum/jupyter>`_ for all
-of Project Jupyter exists, along with one for `teaching with Jupyter
-<https://groups.google.com/forum/#!forum/jupyter-education>`_. 
+There is a `mailing list <https://groups.google.com/forum/#!forum/jupyter>`_ for `teaching with Jupyter
+<https://groups.google.com/forum/#!forum/jupyter-education>`_ as well as one for the entire of Project Jupyter.
```

Comment:
I wasn't aware of these :smile:. Are they still monitored by JupyterHub team members?

Reply:
I do see that the team still has incoming mails on the same but there's no response.
I am also not sure if this is the only place that the mailing list is mentioned. Also, since the team is more active on Discosure and Gitter... do you think we should remove this part?

Reply:
I would not list them here. They do exist for community organization and some folks participate (they are more user-community lists than maintainer lists). For the most part, mailing lists have been replaces by the forum once it was created, so let's focus on that.